.github/workflows/ci-components.yml

```diff
@@ -46,7 +51,7 @@ jobs:
 
     - name: Lint
       run: yarn lint
-
+      working-directory: packages/components
```

Comment:
TODO next wk:
- [ ] Failing at lint in CI (but works locally), need to do some debugging next week
- [ ] See if other Components tests run and work when lint fixed
- [ ] See if Percy is set up correctly when lint fixed
- [ ] Talk to @Dhaulagiri about https://github.com/hashicorp/design-system/pull/50 (not sure what you meant about the npm publish of Tokens), sent DM
- [ ] I think https://github.com/hashicorp/design-system/pull/50 can be +1'd and merged, if you want to. Then discuss the `dist/` things 

Could def be more I missed!

Reply:
😠  getting actions to behave in setups like this have been a bit tricky for me. For some reason it's not running `yarn lint` in the same location it did `yarn install` and the output doesn't do much to help us figure out why (at least for me).

I'm not sure if it's related, but it might help to have the yarn install steps match what is in our other ci workflows since those are quite a bit simpler than what we have here.